src/Symfony/Component/Notifier/Receiver/AdminReceiver.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Receiver;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @experimental in 5.0
+ */
+class AdminReceiver extends Receiver implements SmsReceiverInterface
```

Comment:
should the word `Receiver` be replaced by `Recipient` everywhere? It confused me during the review

Reply:
I used Receiver because of the chat notifications which can be a channel (Slack) and not a person (email or phone number).

Reply:
Renamed one